app/code/community/Bolt/Boltpay/Model/BoltOrder.php

```diff
@@ -345,6 +351,80 @@ protected function getCorrectedTotal($projectedTotal, $magentoDerivedCartData)
         return $magentoDerivedCartData;
     }
 
+    /**
+     * Checks if billing address of a quote has all the expected fields.
+     * If not, the address is updated by using the provided shipping address.
+     *
+     * @param Mage_Sales_Model_Quote $quote Quote that is expected to contain both
+     *                                      a billing and shipping address.
+     *
+     * @return bool  true if a correction is made, otherwise false
+     */
+    public function correctBillingAddress($quote)
+    {
+        $billingAddress = $quote->getBillingAddress();
+        $shippingAddress = $quote->getShippingAddress();
+        $wasCorrected = false;
+
+        /////////////////////////////////////////////
+        /// Changes persisted to Magento and Bolt
+        /////////////////////////////////////////////
+        if (
+            !$billingAddress->getStreet() ||
+            !$billingAddress->getCity() ||
+            !$billingAddress->getCountry()
+        )
+        {
+            $oldPrefix = $billingAddress->getPrefix();
```

Comment:
rename this and below to $billingFoo

Reply:
@daisy1754 
I'm not sure what you are asking here as in which variable you are asking be named billingFoo. 
 Also, $billingFoo is a very ambiguous variable name, and very confusing.  If you are talking about using the metasyntactic variable convention FooBar, I will point out in zero places in the entire Magento code place that uses this psuedo variable name and it seems to be a strange convention to be adding here.

I believe that I must be confused with the request here.